src/App.vue

```diff
@@ -101,6 +107,22 @@ export default defineComponent({
       store.dispatch('app/init');
     });
 
+    watch(priceQueryError, () => {
+      if (priceQueryError.value) {
+        addAlert({
+          id: 'price-error',
```

Comment:
it is essential to give an `id` to alerts so that we can remove them later on. This is because we can have multiple alerts, each with its own priority (and only one can be shown at a time).

If an existing `id` is passed to `addAlert` it will override the existing one.